core/src/main/java/org/mule/runtime/core/routing/filters/ExpressionFilter.java

```diff
@@ -81,7 +82,7 @@ public boolean accept(InternalMessage message, Event.Builder builder) {
     }
 
     // TODO MULE-9341 Remove Filters. Expression filter will be replaced by something that uses MuleEvent.
-    Flow flowConstruct = new Flow("", muleContext);
+    Flow flowConstruct = new DefaultFlow("", muleContext);
```

Comment:
I did not think that was required to force the usage of the builder, but I can change them (related to previous comment)